config/jobs/kubernetes-sigs/kueue/kueue-presubmits.yaml

```diff
@@ -0,0 +1,34 @@
+presubmits:
+  kubernetes-sigs/kueue:
+  - name: pull-kueue-test
+    always_run: true
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-scheduling-kueue-test
+      testgrid-tab-name: build
+      description: "Run kueue tests"
+    spec:
+      containers:
+      - image: golang:1.17
+        command:
+        - make
+        args:
+        - test
+  - name: pull-kueue-build-image
+    always_run: true
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-scheduling-kueue-image
```

Comment:
Can we do sig-scheduling-kueue for both?

Reply:
no, see last error
```bash
could not write config: 1 error occurred:
	* found duplicate name after normalizing: (DashboardTab) build
FAIL	k8s.io/test-infra/config/tests/testgrids	3.742s
```

Reply:
but wasn't that because of testgrid-tab-name? (I didn't see the previous version).

Reply:
fixed! yay!